Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -558,13 +559,17 @@ DeprecatedString HTMLHyperlinkElementUtils::get_an_elements_target() const
 bool HTMLHyperlinkElementUtils::get_an_elements_noopener(StringView target) const
 {
     // To get an element's noopener, given an a, area, or form element element and a string target:
+    auto rel = hyperlink_element_utils_rel();
+    auto link_types = rel.view().split_view_if(Infra::is_ascii_whitespace);
 
-    // FIXME: 1. If element's link types include the noopener or noreferrer
+    // 1. If element's link types include the noopener or noreferrer
     // keyword, then return true.
+    if (link_types.contains_slow("noopener"sv) || link_types.contains_slow("noreferrer"sv))
+        return true;
 
-    // FIXME: 2. If element's link types do not include the opener keyword and
+    // 2. If element's link types do not include the opener keyword and
     // target is an ASCII case-insensitive match for "_blank", then return true.
-    if (target.equals_ignoring_case("_blank"sv))
+    if (link_types.contains_slow("opener"sv) && target.equals_ignoring_case("_blank"sv))
```

Comment:
This condition is backwards, "link types **do not** include the opener keyword"
```suggestion
    if (!link_types.contains_slow("opener"sv) && target.equals_ignoring_case("_blank"sv))
```

Reply:
🤦oof my bad